splice_machine/src/main/java/com/splicemachine/derby/stream/function/AbstractFileFunction.java

```diff
@@ -124,20 +123,34 @@ private void writeNullableUTF(ObjectOutput out, String source) throws IOExceptio
 
     @SuppressFBWarnings(value = "REC_CATCH_EXCEPTION",justification = "Intentional")
     public LocatedRow call(List<String> values,BooleanList quotedColumns) throws Exception {
+        int columnID = 0;
+        String columnValue = null;
+        int numofColumnsinTable = 0;
+        int numofColumnsinFile = 0;
+        boolean columnnumbermistmatch = false;
+
         operationContext.recordRead();
         try {
             ExecRow returnRow = execRow.getClone();
-            if(values==null)
+            if (values == null) {
+                columnnumbermistmatch = true;
```

Comment:
The variable columnnumbermismatch is used later in processing the exception in the catch part.